web/src/peer_data.ts

```diff
@@ -211,6 +211,21 @@ export function get_subscribers(stream_id: number): number[] {
     return [...subscribers.keys()];
 }
 
+export async function get_all_subscribers(
+    stream_id: number,
+    retry_on_failure = true,
+): Promise<number[] | null> {
+    // This function parallels `get_subscribers` but ensures we include all
+    // subscribers, possibly fetching that data from the server.
```

Comment:
Is there a reason callers would still want to use the other `get_subscribers`? It's generally good to avoid having two similar APIs with similar names like this, especially if no code should be using one of them.

Reply:
More details [in this doc](https://paper.dropbox.com/doc/trimming-subscriber-data--Cl1vS0RcK4sSALl7xrO7z495Ag-yQnHu9K61oiJaDZhdqR4y) but there are three calls to `get_subscribers` on `main` right now and I'm transitioning them one by one, so if we do deduplicate then I imagine that would happen then.

Pills that bulk add users will need to fetch, but the buddy list would probably want to use the non-fetch version of `get_subscribers`.

Reply:
OK. We should make sure the plan in the doc includes doing a full pass on this doc around `loaded` being in the name.

Reply:
Added to the doc!